tests/app/data/test_schema_validation.py

```diff
@@ -83,8 +75,6 @@ def validate_json_against_schema(file, json_to_validate, schema):
             return []
         except jsonschema.exceptions.ValidationError as e:
             return ["Schema Validation Error! File [{}] does not validate against schema. Error [{}]".format(file, e)]
-        except JSONDecodeError as e:
```

Comment:
In fact, I'd argue that if something else fails here (other than schema validation) then we're unlikely to be able to recover from it, so not sure what benefits a catch-all exception would provide.